docs/README.md

```diff
@@ -1,11 +1,11 @@
-
-
 📢 Use this project, [contribute](https://github.com/vtex-apps/search-result) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

Comment:
Please revert unnecessary changes in this file

Reply:
Reverted the prettified readme